mips_api/__init__.py

```diff
@@ -229,14 +263,14 @@ def _build_return(code, body):
             event_path = event['path']
 
             if event_path == api_routes['ApiChartOfAccounts']:
-                # return chart of accounts
+                # return chart of accounts, optionally processed
+                mips_chart = filter_chart(params, raw_chart, omit_codes_list, extra_codes_dict)
                 return _build_return(200, mips_chart)
 
             elif event_path == api_routes['ApiValidTags']:
-                try:
-                    valid_tags = list_tags(params, mips_chart, omit_codes_list, extra_codes_dict)
-                except Exception as exc:
-                    return _build_return(500, {"error": str(exc)})
+                # always process the chart for the tags list
+                mips_chart = process_chart(raw_chart, omit_codes_list, extra_codes_dict)
+                valid_tags = list_tags(params, mips_chart)
```

Comment:
currently the `limit` option is only in list_tags.  should it not be usable for filter_chart as well?

Reply:
I hadn't thought of that, but yeah, I'll add it for completeness